### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by ljukas**
Dynamic menu now working. Pulls content from the database. Atm it uses a local database.

Has some leftover styling from using a second sliding side-nav.

### CODE REVIEW

File: app/database/response.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module HTTParty
+  class Response < BasicObject
+
+    # HTTParty response objects lack an easy way to check if the request succeeded,
+    # thus we extend it with this functionality ourselves
+    def success?
+      response.code >= 200 && response.code <= 299
```
Comment (patriksletmo) [inline app/database/response.rb]: Errors in production environment.

File: app/database/response.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module HTTParty
+  class Response < BasicObject
```
Comment (patriksletmo) [inline app/database/response.rb]: Class extension not necessary as `HTTParty::Response` already contains a definition for `success?`.

File: app/assets/javascripts/_navigation.js [hunk 3]
```diff
@@ -2,65 +2,14 @@
...
-        sidenavActive = setTimeout(function () {
-            if ($('#secondary-nav:visible')) {
-                navcontrol.sideNav('hide');
-            }
-        }, 200);
-        sidenavVisibility = setTimeout(function () {
-            $('#secondary-nav').addClass('hide');
-        }, 300);
-    });
+    // Initialize collapsible (uncomment the line below if you use the dropdown variation)
```
Comment (patriksletmo) [inline app/assets/javascripts/_navigation.js]: Consider rephrasing comment.

File: app/assets/stylesheets/colors.scss [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
+$primary-color: #e74c4c;
+$secondary-color: #9b59b6;
+$accent-color: #F5D76E;
```
Comment (patriksletmo) [inline app/assets/stylesheets/colors.scss]: Very minor issue but please ensure hex colors follow the same case.

File: app/assets/javascripts/_navigation.js (hunk 3 above)
Comment (ljukas) [inline app/assets/javascripts/_navigation.js]: Removed parathesis part of the comment

File: app/assets/stylesheets/colors.scss (hunk 4 above)
Comment (ljukas) [inline app/assets/stylesheets/colors.scss]: Reformated to downcase

**Comment by patriksletmo** [comment]
Ready for merge. Please wait until pull request in db-app has been merged.

**Pull request merged**